cms/src/main/java/uk/nhs/digital/apispecs/handlebars/EnumHelper.java

```diff
@@ -9,11 +9,17 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
-public class EnumHelper implements Helper<Collection<String>> {
+public class EnumHelper implements Helper<Collection<?>> {
 
-    public static final String NAME = "enum";
+    public static final EnumHelper INSTANCE = new EnumHelper();
 
-    @Override public Object apply(final Collection<String> args, final Options options) {
+    public static final String NAME = "enumeration";
```

Comment:
The helper's name was changed from `enum` to `enumeration` to eliminate confusion: the relevant field in JSON is called `enum` and it could look puzzling to see the helper invoked in the template as `{{{enum enum}}}`.

Reply:
Good idea.